pkg/netns/netns_linux.go

```diff
@@ -150,6 +169,11 @@ func NewNS() (ns.NetNS, error) {
 
 // UnmountNS unmounts the NS held by the netns object
 func UnmountNS(ns ns.NetNS) error {
+	nsRunDir, err := getNSRunDir()
```

Comment:
Hm. Where is this used? It seems like it ought to be complaining that this is unused...

Reply:
https://github.com/containers/libpod/blob/1bbcb2fc56c1655269930c83628ef5f5802f1887/pkg/netns/netns_linux.go#L155